matrixprofile/compute.py

```diff
@@ -61,6 +63,10 @@ def compute(ts, windows=None, query=None, sample_pct=1, threshold=0.98,
         window(s) is given.
     n_jobs : int, default = 1
         Number of cpu cores to use.
+    preprocessing_kwargs : dict, default = None
```

Comment:
Please be more verbose about what the preprocessing kwargs are. This will make it easier for the end-users to know what to do via the API documentation.